rentals.json

```diff
@@ -0,0 +1 @@
+[{"book":null,"person":null},{"date":"2020/07/11","book":"770 By ","person":359},{"date":"2022/05/04","book":" By ","person":787}]
```

Comment:
@SiansandaTamara as previously requested kindly remove the first argument, you can set your default person to empty string so that it doesn't display null.

Reply:
@danmainah , I have already removed the first argument which was date